### ISSUE TITLE
Unexpected behavior with 64 length connectors 

### ISSUE THREAD

**Issue by armstrys** [holesail/holesail#5]
I am getting different behavior responses from holesail with 64 length connectors. See 3 example cases below:

Case 1 - 64 character cryptographic hash
```
holesail --live xxxx --host xxxxx --connector fa19d15821658c1fc31f600ebc922e2ad5ab0d5bd0fb534392de6d5759d4f9a6
```
yields

```
Notice: There is no connector set. 

Server started, Now listening on xxxxx:xxxxx
Server public key: 27c237............d369
```

Case 2 - 64 xs:
```
holesail --live xxxx --host xxxxx --connector xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

gives

```
/Users/armstrys/.nvm/versions/node/v20.14.0/lib/node_modules/holesail/node_modules/hyperdht/lib/crypto.js:13
  if (seed) sodium.crypto_sign_seed_keypair(publicKey, secretKey, seed)
                   ^

Error: "seed" must be crypto_sign_SEEDBYTES bytes long
    at createKeyPair (/Users/armstrys/.nvm/versions/node/v20.14.0/lib/node_modules/holesail/node_modules/hyperdht/lib/crypto.js:13:20)
    at HyperDHT.keyPair (/Users/armstrys/.nvm/versions/node/v20.14.0/lib/node_modules/holesail/node_modules/hyperdht/index.js:401:12)
    at holesailServer.keyPairGenerator (/Users/armstrys/.nvm/versions/node/v20.14.0/lib/node_modules/holesail/node_modules/holesail-server/index.js:30:29)
    at holesailServer.serve (/Users/armstrys/.nvm/versions/node/v20.14.0/lib/node_modules/holesail/node_modules/holesail-server/index.js:39:10)
    at Object.<anonymous> (/Users/armstrys/.nvm/versions/node/v20.14.0/lib/node_modules/holesail/index.js:57:17)
    at Module._compile (node:internal/modules/cjs/loader:1358:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
```

Case 3 - 64 1s 

```
holesail --live xxxx --host xxxxx --connector 1111111111111111111111111111111111111111111111111111111111111111
```

works as expected.

**Comment by supersuryaansh** [comment]
@armstrys a 64 length connector is considered a key. There is a length check in the module if the given input is of length 64 then it is a key any length except 64 is a connector.

**Comment by supersuryaansh** [comment]
Yes thats correct, avoid 64 character connectors.

**Comment by supersuryaansh** [comment]
@armstrys It is actually --connector, you need to use this parameter to start a private DHT. EXAMPLE:

holesail --live 5050 --host localhost --connector my-word-bool-quite-max-king

This will start a Private DHT.

I am currently working on pushing a new update to holesail which will clarify the commands and make it easier to understand.

**Comment by supersuryaansh** [comment]
@armstrys Ohh I see, yeah that behaviour will be fixed soon in a planned security update.

I just pushed out an UI update today which highlights what mode the user is using so that will make some difference as well, make sure you are on the latest version.

**Comment by supersuryaansh** [comment]
@armstrys Thanks, I am aware of this issue; it has to do with data validation. I will be pushing a new minor update soon to fix this and other small bugs.

**Comment by supersuryaansh** [comment]
@armstrys This is now fixed in version 1.5.1 🎉
